packages/react-components/react-button/src/components/CompoundButton/useCompoundButtonStyles.ts

```diff
@@ -118,19 +118,24 @@ const useRootStyles = makeStyles({
 
   // Size variations
   small: {
-    ...shorthands.padding('8px', '8px', '10px', '8px'),
+    ...shorthands.padding(
+      tokens.spacingHorizontalS,
+      tokens.spacingHorizontalS,
+      tokens.spacingHorizontalMNudge,
+      tokens.spacingHorizontalS,
+    ),
 
     fontSize: tokens.fontSizeBase300,
     lineHeight: tokens.lineHeightBase300,
   },
   medium: {
-    ...shorthands.padding('14px', '12px', '16px', '12px'),
+    ...shorthands.padding('14px', tokens.spacingHorizontalM, tokens.spacingHorizontalL, tokens.spacingHorizontalM),
```

Comment:
This seems wrong to mix and match hardcoded values with tokens. Is there a bug with the design that it needs to be updated to use tokens here? Or is the `14px` value calculated as the sum of two tokens? If so, use `calc()` instead.